src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -817,4 +821,4 @@ testLoadingVector = Loading test vector “%s”…
 testResults = Passed: %s, Failed: %s
 testRunning = Running %s vectors…
 testSetupFailed = Error preparing test vector: %s
-untitled = untitled
+untitled = untitled
```

Comment:
Why is the end of this file modified? Only do so if a newline is missing at the end of the last line.